modules/assim.sequential/inst/paleon_sda.R

```diff
@@ -1,26 +1,49 @@
 
 library(PEcAn.all)
 library(PEcAn.SIPNET)
+library(PEcAn.LINKAGES)
 library(PEcAn.assim.sequential)
+library(nimble)
+library(lubridate)
+
+ciEnvelope <- function(x,ylo,yhi,...){
+  polygon(cbind(c(x, rev(x), x[1]), c(ylo, rev(yhi),
+                                      ylo[1])), border = NA,...) 
+}
```

Comment:
I don't think you need to redefine this since it's exported by the visualization module, which should load when you load PEcAn.all (if not, that's a bug)